assets/keymaps/default-macos.json

```diff
@@ -190,6 +190,12 @@
       "ctrl-shift-enter": "editor::NewlineBelow"
     }
   },
+  {
+    "context": "Editor && copilot_suggestion",
```

Comment:
https://github.com/zed-industries/zed/blob/e6bec9a24a12773fc8e9d4ed0a4d37cb06bed75c/assets/keymaps/default-linux.json#L138-L140

We seem to have a group of copilot keybindings, starting with `alt` and `alt-right` is vacant, so let's regroup the whole thing?

Seems that existing copilot keybindings should work in any `Editor` context, and only `copilot::Suggest` should have `!copilot_suggestion`, we can create another block for the `copilot::Suggest` and move the rest of the existing keybindings here.

Let's also change the `cmd` to `alt` to match the rest of the bindings and do the same changes to `default-linux.json`

Reply:
Seems like a good idea. But right now alt-right is used for moving to the end of a word. Maybe we should do the blocks like this:
```
{
  "context": "Editor && copilot_suggestion",
  "bindings":
      "alt-]": "copilot::NextSuggestion",
      "alt-[": "copilot::PreviousSuggestion",
      "alt-right": "editor::AcceptPartialCopilotSuggestion",
    }
  },
  {
    "context": "Editor && !copilot_suggestion",
    "bindings": {
      "alt-\\": "copilot::Suggest"
    }
  }
```

Reply:
Right, how did I manage to miss the `MoveToNextWordEnd`? 😕 
Still, looks like it works just fine in the context I've proposed, and we do not propose copilot suggestions that are not on the "edge" of a block, so seems ok to keep it this way maybe.

We can change it later if somebody comes with a good example that breaks the workflow.